### ISSUE TITLE
Add support to specify EPUB Accessibility and WCAG Version and Level

### ISSUE THREAD

**Issue by gregoriopellegrino** [daisy/ace#390]
Currently, Ace does not provide an option to specify the version of EPUB Accessibility specifications, the version of WCAG (Web Content Accessibility Guidelines), or the desired WCAG level when checking an EPUB file for accessibility compliance. This can be limiting for users who want to ensure that their EPUB files adhere to specific accessibility standards and levels. This proposal suggests enhancing Ace to include these specifications as arguments for more precise accessibility checks.

## Proposal

- Add a new argument, that allows users to specify the version of EPUB Accessibility specifications they want to use as the base for accessibility checks. This can be set to either "1.0" or "1.1."
- Introduce a new argument, to enable users to specify the version of WCAG they want to reference during accessibility checks. This can accept values like "2.0," "2.1," and, in the future, "2.2."
- Add another argument, to allow users to specify the desired WCAG level for the accessibility checks. This argument can accept values such as "A," "AA," or "AAA."

In my opinion, by default, these arguments should be set to the latest version of EPUB Accessibility and WCAG 2.x, level AA.

**Comment by danielweck** [comment]
FYI, we are currently integrating the latest version of Deque Axe which is used by Ace to perform the checks inside HTML documents.

https://github.com/dequelabs/axe-core/releases/tag/v4.9.1

https://github.com/daisy/axe-core/pull/12

WCAG support is described here:

https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md

Note that the EPUB Accessibility specification currently only references 2.1, but this will be tracked upwards as the standards evolve:

https://w3c.github.io/epub-specs/epub33/a11y/#sec-conf-reporting-pub

The architectural changes required to make Ace configurable in terms of the level of support for a set of revisions of particular specifications will be significant, and technically there is only so much Deque Axe allows us (the integrators / API consumers) to configure. Still, definitely worth looking into. Thank you for your suggestions.

**Comment by danielweck** [comment]
Related: https://github.com/daisy/ace/issues/422
